README.md

```diff
@@ -3,11 +3,16 @@ A collection of Cyclus manager archetypes for demand driven deployment. It opera
 global calls within a Cyclus simulation so that all agents within the simulation
 can communicate. The goal of this package is to provide three types of mathematical
 basis for determining supply and demand of commodities within Cyclus; Non-optimizing (NO)
-deterministing optimization (DO), and Stochastic optimization (SO). 
+deterministing optimization (DO), and Stochastic optimization (SO).
+
+The user needs to specify the supply chain by setting the appropriate `initialfacilitylist`. With the
```

Comment:
Perhaps: The user needs to specify the prototypes in the simulation by defining the facilities in the input file and including one of each facility in the `initialfacilitylist`.

Reply:
I don't think they should have to be in the intitalfacilitylist. Right now they are, but d3ploy doesn't read that. It uses the prototypes list. In the end I think @gwenchee  change to add the prototype entry will make the initialfacilitylist completely disconnected from the actual prototypes tracked.

Reply:
My understanding was that we needed two things:
1. some sort of 'initial condition' so that d3ploy know what prototype has how much capacity
2. the supply chain (and what prototypes are involved)
I think having an initialfacility of one-of-each of every prototype in the supply chain will accomplish both, without additional input for NOInst. In the beginning, the NOInst would listen to the timeseries recorded by its children and construct the supply chain, as well as prototypes invovled. It will also know (somewhat) what the capacity of each prototype is, so that it can deploy according to scale.

Reply:
Except in the case that a facility does not exist at the start of simulation. For example, a recycle scenario may not start until the 40th year. In which cause, having a separations facility and a fast reactor in the initial facilities list will be incorrect. 

It's important to look forward and try not to be too clever with solutions to might make things awkward later on.

Reply:
ah. good point. So how do we find out the capacity of the prototypes without the initial conditions?

Reply:
That's the part that @gwenchee is working on at the moment. She's implemented a system we discussed to feed prototypes in with the information required.